### PR TITLE
Blog app - setup and controllers

### PR DESCRIPTION
**Pull request by lfmnovaes**
- Initiated RoR project with postgresql database
- Added rubocop and stylelint linters
- Run `rails scaffold` to create initial routing, controller, and views

### CODE REVIEW

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
Hi @lfmnovaes  

You've done well implementing some of the requested changes, but there are still some which aren't addressed yet, please work on it and ask for another review.


 ## Required Changes ♻️
- The `rails scaffold` is not recommended for this project, kindly use another way to implement this milestone and it will be used later for other milestones instead of using scaffold because scaffold creates everything for you.

_Check the comments under the review._

## Optional suggestions
- n/a

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
You can DM or reach out to me on slack [github](https://github.com/Emmyn5600) or [twitter](https://twitter.com/NSABIMA62253884)

File: config/routes.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Rails.application.routes.draw do
+  resources :users do 
+    resources :posts
```
Comment (eliasecasta) [inline config/routes.rb]: - [ ] Please do not expose unnecessary endpoints, for now, it is a requirement to have just index and show exposed for both, users and posts, please do so to comply with the project requirements, everything else is good to go 😉

**Review by eliasecasta — COMMENTED** [review]
Hello @lfmnovaes 👋,

_This is an additional review as a result of the second opinion request._

You are correct about the scaffold usage, it is not prohibited, @Emmyn5600 please stick to the project guidelines in the future and check only what is needed. I do agree that it is a better learning way to manually create everything and I highly recommend you @lfmnovaes to do so, but there is no rule stopping you from using the scaffold command.

I performed a full review just in case, please check the routes endpoints comment that I left and after that, your project should be approved without other issues.

Please ask for another review in your Dashboard and the next Code Reviewer will mark it as approved right away after the issue that I mentioned you is addressed :)

Keep rocking 🚀!

**Review by addod19 — APPROVED** [review]
### Hi @lfmnovaes, 

Your project is complete!  Nice job fixing the exposed routes :fireworks: There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**